app/Providers/PayPalServiceProvider.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Providers;
+
+use Illuminate\Support\ServiceProvider;
+
+class PayPalServiceProvider extends ServiceProvider
+{
+    /**
+     * Bootstrap the application services.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        //
+    }
+
+    /**
+     * Register the application services.
+     *
+     * @return void
+     */
+    public function register()
+    {
+        // Add paypal library as singleton
+        $this->app->singleton(PayPalServiceProvider::class, function($app){
+            return new \PayPal\Rest\ApiContext(
+                new \PayPal\Auth\OAuthTokenCredential(
+                    'ATPZcJYf7Ob4M9pH4CSFbpVhqwxW5HH1rntthIiSETmQMQWhDgDDS-1UBP1_PDBlHNjDKaV_8nLyw-3b',     // ClientID
```

Comment:
Please used `config('services.paypal.client_id')` and `config('services.paypal.client_secret')`. 
You are exposing API keys on Github.